umodbus/sys_imports.py

```diff
@@ -0,0 +1,29 @@
+try:
```

Comment:
please add a shebang and header as done in the other files

Reply:
Good catch, will add it